source/_integrations/thread.markdown

```diff
@@ -11,5 +11,12 @@ ha_integration_type: service
 ha_config_flow: true
 ---
 
-The Thread integration is currently a placeholder, but will manage Thread network credentials in a future release.
-The integration is automatically setup when the "Silicon Labs Multiprotocol" add-on is installed.
+The Thread integration helps you track the different Thread networks in your home, the credentials that they use and helps you align them. It is currently a work in progress.
+
+Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. Thread allows devices to communicate with each other without the need for a central gateway or router. This makes it ideal for home automation, where a large number of devices may need to be connected.
+
+To connect a Thread network to Home Assistant, Thread Border Routers (TBRs) are used. TBRs are devices that bridge the Thread network to a local wi-fi or ethernet network. To add a TBR to Home Assistant, you can use our [Home Assistant Yellow](/yellow/) hub or the [Home Assistant SkyConnect](/skyconnect/) Zigbee/Thread stick.
+
+Home Assistant communicates with TBRs over the local network. This means that TBRs do not have to be physically attached to Home Assistant to be used. You might already have a TBR as part of other products in your home, like a Google Nest Hub Gen 2 or Apple HomePod Mini. Each vendor forms their own network when you start using their product, so you can end up having a Home Assistant, Apple and Google Thread networks in your home. These are all separate networks using different credentials, which prevents devices to roam between TBRs.
```

Comment:
Each vendor forms their own network when you start using their products, so you can end up having a Home Assistant, an Apple, and a Google Thread network in your home.